MovingBlocks/Terasology
# [telemetry] game play metric

### Contains

Game play metric including:

- Block destroyed
- Block placed
- Monster killed
- Game configuration: word generator, network mode, language, player height, player eye height
- Game play stats: distance traveled, play time

### How to test

- In a game, the stats will show in the `Metrics Menu`
- If you enable `Send metrics to server`, the metrics will be sent to http://utility.terasology.org/. You can see on port 5601 http://utility.terasology.org:5601/

![gameplay](https://user-images.githubusercontent.com/17897358/28490193-50a4f8a4-6f06-11e7-8b67-5ac901fd51f8.png)
![gameconfiguration](https://user-images.githubusercontent.com/17897358/28490191-50a07e8c-6f06-11e7-9010-da02fb865f58.png)
![blockdestroyed](https://user-images.githubusercontent.com/17897358/28490192-50a0e9ee-6f06-11e7-8265-92429c82b5f5.png)

@rzats @oniatus @skaldarnar @msteiger @Cervator @qwc thanks for review :-)
